app/react-native/package.json

```diff
@@ -56,7 +56,7 @@
     "json-loader": "^0.5.4",
     "json5": "^0.5.1",
     "postcss-loader": "^2.0.5",
-    "react-native-compat": "0.0.2",
+    "react-native-compat": "0.0.4",
```

Comment:
do you plan on versioning according to semvar? we could do `~0.0.4` so we don't have to do a PR for every minor fix.